src/libs/auth.js

```diff
@@ -93,8 +93,11 @@ window.forceSignIn = async token => {
 
 authStore.subscribe(async (state, oldState) => {
   if (!oldState.isSignedIn && state.isSignedIn) {
+    const isTrustedEmail = (state.user.email.match(/@.*/g) === '@broadinstitute.org') ||
```

Comment:
Does this work? I think `match` returns an array, not a string.

Reply:
Shoot, you're right! Would changing the conditional so that the email strings were single element arrays suffice (feels like a bad solution, but gets the job done)? Or should I try to find another function that is similar to match but returns a string?